File: 9-Faktöriyel/Faktoryel.cpp
#include <iostream>

using namespace std;
int main()
{
    // “i”, “f”, “n” tanımlanması
    int i, n, f = 1;
    cout << "n=";
  
    // “n”nin değerinin girilmesi
    cin >> n;

    // n!’i hesaplayan döngünün oluşturulması
    // “i=1” olduğunda sonucu etkilemeyeceği için “i=2”den başlıyoruz
    for (i = 2; i <= n; i++)

        // “f” değerini i sayacının değerleriyle çarpacak işlem
        f = f * i; 

    // “f”nin değerinin yazdırılması
    cout << "n!=" << f;
 }

File: 9-Faktöriyel/Faktoryel.py
# “i”, “f”, “n” tanımlanması
n = 0
i = 0
f = 1

# “n”nin değerinin girilmesi
n = int(input("n= "))

# n!’i hesaplayan döngünün oluşturulması.
# “i=1” olduğunda sonucu etkilemeyeceği için “i=2”den başlıyoruz.
# range() foksiyonun özelliğinden dolayı range fonsiyonu bitiş değeri
# n+1 olarak vereceğiz.
for i in range(2,n+1):
    # “f” değerini i sayacının değerleriyle çarpacak işlem
    f = f * i

# “f”nin değerinin yazdırılması
print("n!=",f)
